{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"wrap"},{"kind":"text","text":"("},{"kind":"externalParam","text":"_"},{"kind":"text","text":" "},{"kind":"internalParam","text":"error"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":") -> "},{"kind":"typeIdentifier","identifier":"doc:\/\/MGNetworking\/documentation\/MGNetworking\/NetworkClientError","preciseIdentifier":"s:12MGNetworking18NetworkClientErrorO","text":"NetworkClientError"}],"languages":["swift"],"platforms":["macOS"]}]},{"kind":"content","content":[{"anchor":"return-value","level":2,"type":"heading","text":"Return Value"},{"type":"paragraph","inlineContent":[{"type":"text","text":"The network client error case corresponding to "},{"type":"codeVoice","code":"error"},{"type":"text","text":"."}]}]},{"kind":"parameters","parameters":[{"name":"error","content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"A general error."}]}]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/mgnetworking\/networkclienterror\/wrap(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/MGNetworking\/documentation\/MGNetworking\/NetworkClientError\/wrap(_:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Returns an appropriate network client error for the passed error."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"wrap"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":") -> "},{"kind":"typeIdentifier","text":"NetworkClientError","preciseIdentifier":"s:12MGNetworking18NetworkClientErrorO"}],"title":"wrap(_:)","roleHeading":"Type Method","role":"symbol","symbolKind":"method","externalID":"s:12MGNetworking18NetworkClientErrorO4wrapyACs0D0_pFZ","modules":[{"name":"MGNetworking"}]},"hierarchy":{"paths":[["doc:\/\/MGNetworking\/documentation\/MGNetworking","doc:\/\/MGNetworking\/documentation\/MGNetworking\/NetworkClientError"]]},"references":{"doc://MGNetworking/documentation/MGNetworking/NetworkClientError/wrap(_:)":{"role":"symbol","title":"wrap(_:)","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"wrap"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":") -> "},{"kind":"typeIdentifier","text":"NetworkClientError","preciseIdentifier":"s:12MGNetworking18NetworkClientErrorO"}],"abstract":[{"type":"text","text":"Returns an appropriate network client error for the passed error."}],"identifier":"doc:\/\/MGNetworking\/documentation\/MGNetworking\/NetworkClientError\/wrap(_:)","kind":"symbol","type":"topic","url":"\/documentation\/mgnetworking\/networkclienterror\/wrap(_:)"},"doc://MGNetworking/documentation/MGNetworking/NetworkClientError":{"role":"symbol","title":"NetworkClientError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"NetworkClientError"}],"abstract":[{"type":"text","text":"A network error."}],"identifier":"doc:\/\/MGNetworking\/documentation\/MGNetworking\/NetworkClientError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"NetworkClientError"}],"url":"\/documentation\/mgnetworking\/networkclienterror"},"doc://MGNetworking/documentation/MGNetworking":{"role":"collection","title":"MGNetworking","abstract":[],"identifier":"doc:\/\/MGNetworking\/documentation\/MGNetworking","kind":"symbol","type":"topic","url":"\/documentation\/mgnetworking"}}}